README.md

```diff
@@ -1,15 +1,111 @@
-# Sourceloop Angular Boiler Plate
+#### ANGULAR BOILERPLATE
+[![Version](https://img.shields.io/badge/@angular/core-v14-brightgreen)](http://commitizen.github.io/cz-cli/)
 
-### Feedback
+<!-- DOCUMENTATION -->
 
-If you've noticed a bug or have a question or have a feature request, [search the issue tracker]([Issues · sourcefuse/angular-boilerplate · GitHub](https://github.com/sourcefuse/angular-boilerplate/issues)) to see if someone else in the community has already created a ticket. If not, go ahead and [make one](https://github.com/sourcefuse/angular-boilerplate/issues/new/choose)! All feature requests are welcome. Implementation time may vary. Feel free to contribute the same, if you can. If you think this extension is useful, please [star](https://help.github.com/en/articles/about-stars) it. Appreciation really helps in keeping this project alive.
+# Description
 
-### Code of Conduct
+- The boilerplate using multiple projects in one Angular environment so that the boilerplate help users 
+  to improve code  reusability, scalability, maintainability, and customization. It can also help in increase productivity, reduce risks, and improve the quality of  application.
 
-Code of conduct guidelines [here.](CODE_OF_CONDUCT.md)
+- The multiple applications or parts of applications that share common functionality,  using multiple
+  projects can help you avoid duplicating code. You can create a shared library project and use it across all the projects. This approach can help you maintain consistency and reduce code duplication.
```

Comment:
Change this description content the current one doesn't make much sense.
Also description is never a bulleted list.